pkg/kubelet/dockershim/libdocker/helpers.go

```diff
@@ -60,6 +60,13 @@ func matchImageTagOrSHA(inspected dockertypes.ImageInspect, image string) bool {
 			// hostname or not, we only check for the suffix match.
 			if strings.HasSuffix(image, tag) || strings.HasSuffix(tag, image) {
 				return true
+			} else {
+				t, _ := dockerref.ParseNormalizedNamed(tag)
+				t2 := t.(dockerref.Tagged)
```

Comment:
check the cast `t2, ok := t.(dockerref.Tagged)` and continue if !ok

Reply:
Done!